nekoyume/Assets/_Scripts/Game/Stage.cs

```diff
@@ -622,7 +628,12 @@ private IEnumerator CoSlideBg()
 
         private IEnumerator CoRankingBattleEnd(BattleLog log, bool forceQuit = false)
         {
+            ActionRenderHandler.Instance.Pending = false;
             _onEnterToStageEnd.OnNext(this);
+            yield return ActionRenderHandler.Instance
+                .AvatarStateForBattleResultSubject
+                .First();
+
```

Comment:
```suggestion
            yield return ActionRenderHandler.Instance
                .AvatarStateForBattleResultSubject
                .DoOnSubscribe(() =>
                {
                    ActionRenderHandler.Instance.Pending = false;
                    _onEnterToStageEnd.OnNext(this);
                })
                .First();
```

Reply:
I think this subject not subscribed here, but just waiting for message. Will those lines work?